orm/src/test/java/org/hibernate/search/test/configuration/IndexManagerOverrideTest.java

```diff
@@ -27,14 +27,18 @@
  */
 public class IndexManagerOverrideTest {
 
+	public static class FooIndexManager extends DirectoryBasedIndexManager {
+
+	}
+
 	@Test
 	public void verifyIndexExclusivity() {
 		FullTextSessionBuilder builder = new FullTextSessionBuilder();
 		FullTextSession ftSession = builder
 			.setProperty( "hibernate.search.Book.indexmanager", "near-real-time" )
 			.setProperty(
 					"hibernate.search." + Foo.class.getName() + ".indexmanager",
-					"org.hibernate.search.testsupport.indexmanager.RamIndexManager"
+					FooIndexManager.class.getName()
```

Comment:
`RamIndexManager`'s constructor became private because it now has to be instanciated through `RamIndexManagerFactory`, so that it can be destroyed after the tests. Thus it could not be used here anymore.
I though I could change this because, if I understood this test correctly, the actual class of the index manager does not matter here, we're just checking that Hibernate Search instantiates the type we asked for. So, `RamIndexManager` or not, it does not matter.
Besides, **if** we did rely on some feature of `RamIndexManager` here, there probably was something wrong, since Hibernate Search didn't call `RamIndexManager.makeRamDirectory` but simply the public default constructor, which skips most of the manager configuration.